### PR TITLE
Added support for ware 2nd gen.

### PR DESCRIPTION
**Pull request by dingusdk**
This pull request is support for The Airthings Wave 2nd generation. This new version or the airthings wave has a new uuid, but returns data the same way as the wave plus (but only a subset)

### CODE REVIEW

File: custom_components/airthings_wave/airthings.py [hunk 1]
```diff
@@ -86,6 +87,21 @@ def decode_data(self, raw_data):
         return data
 
 
+class Wave2Decode(BaseDecode):
+    def decode_data(self, raw_data):
+        val = super().decode_data(raw_data)
+        val = val[self.name]
+        # if val[0] != 1:
```
Comment (MartyTremblay) [inline custom_components/airthings_wave/airthings.py:94]: Just being a stickler here. Should this dead code be removed? Any reason to keep it in a comment?

Comment (dingusdk) [reply custom_components/airthings_wave/airthings.py:94]: Yes it can be removed, i just forgot.

**Comment by dingusdk** [comment]
I have verified it on my airthings wave 2. The uuid i got from airthigs example: 
https://github.com/Airthings/wave-reader/blob/master/read_wave2.py

**Comment by MartyTremblay** [comment]
Added support for ware 2nd gen

**Comment by MartyTremblay** [comment]
resolves #23

**Pull request merged**